consensus/abci.go

```diff
@@ -94,127 +89,110 @@ func (abci *TendermintABCI) Info(req tmtAbciTypes.RequestInfo) tmtAbciTypes.Resp
 
 // SetOption sets a configuration option
 func (abci *TendermintABCI) SetOption(req tmtAbciTypes.RequestSetOption) tmtAbciTypes.ResponseSetOption {
-	abci.logger.Info("TendermintABCI::SetOption()")
 	return tmtAbciTypes.ResponseSetOption{Code: tmtAbciTypes.CodeTypeOK, Log: ""}
 }
 
 // InitChain initializes the validator set
 func (abci *TendermintABCI) InitChain(req tmtAbciTypes.RequestInitChain) tmtAbciTypes.ResponseInitChain {
-	abci.logger.Info("TendermintABCI::InitChain()")
 	return tmtAbciTypes.ResponseInitChain{}
 }
 
 // CheckTx checks a transaction is valid but does not mutate the state
 func (abci *TendermintABCI) CheckTx(txBytes []byte) tmtAbciTypes.ResponseCheckTx {
-	abci.logger.Info("TendermintABCI::CheckTx()")
 	tx, err := decodeRLP(txBytes)
 	if err != nil {
-		// nolint: errcheck
 		abci.logger.Error("Received invalid transaction", "tx", tx.Hash().String())
 		return tmtAbciTypes.ResponseCheckTx{Code: uint32(abciTypes.ErrEncodingError.Code), Log: err.Error()}
 	}
 
-	abci.logger.Info("Received valid transaction", "tx", tx.Hash().String()) // nolint: errcheck
 	return abci.validateTx(tx)
 }
 
 // DeliverTx executes a transaction against the latest state
 func (abci *TendermintABCI) DeliverTx(txBytes []byte) tmtAbciTypes.ResponseDeliverTx {
-	abci.logger.Info("TendermintABCI::DeliverTx()")
```

Comment:
Restore as `Debug()`

Reply:
was not removed, better logging with even more info was added